tools/template_manager.py

```diff
@@ -1,14 +1,17 @@
-#!/usr/bin/env python3
 from jinja2 import Template
-import argparse
 from os import path
 from os.path import isfile
+
```

Comment:
Needs docstrings please :-)

Reply:
I won't be adding any to this other file, this is an internal API that should never need to be interacted with specifically during normal validation procedures.